### PR TITLE
Fix iOS status bar style message for Hidden value

### PR DESCRIPTION
**Pull request by TsvetanMilanov**
The Hidden value for iOS status bar style is supported only on iOS versions lower than iOS 7.

### CODE REVIEW

File: lib/project.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ export class Project implements Project.IProject {
 		"kendoui.blank": "KendoUI.Empty",
 		"javascript.blank": "Blank"
 	};
+	private static WRONG_IOS_STATUS_BAR_DESCRIPTION = "Hidden";
```
Comment (nirinchev) [inline lib/project.ts]: The name of this field seems off - there's nothing wrong or descriptive about the word "Hidden" 😺

Comment (rosen-vladimirov) [inline lib/project.ts]: I Agree, `HIDDEN_IOS_STATUS_BAR_DESCRIPTION` seems better

**Comment by rosen-vladimirov** [comment]
So, if I use `appbuilder prop set iOSStatusBar "Hidden (iOS 6.1 and earlier)"`, as I've seen it from `$ appbuilder prop print iOSStatusBar --validValues`, will it work for me ?

**Comment by TsvetanMilanov** [comment]
Closing because Cordova 4.0 has minimum deployment target iOS 8.0 and this bug is for versions earlier than iOS 6.1,  it does not need to be fixed.

**Pull request closed without merging**